src/main/kotlin/uk/gov/justice/digital/hmpps/pecs/jpc/domain/move/MovePersister.kt

```diff
@@ -120,27 +118,31 @@ class MovePersister(
 
   fun processJourneys(move: Move, journeys: List<Journey>, journeyEvents: List<Event>): List<Journey> {
     return if (move.moveType() == MoveType.CANCELLED && journeys.isEmpty())
-      listOf(fakeCancelledJourneyForPricing(move)) // add a fake cancelled journey
+      listOf(fakeCancelledJourneyForPricing(move))
     else
       journeys.map { journey ->
         val thisJourneyEvents = journeyEvents.filter { it.eventableId == journey.journeyId }
         val pickUp = Event.getLatestByType(thisJourneyEvents, EventType.JOURNEY_START)?.occurredAt
         val dropOff = Event.getLatestByType(thisJourneyEvents, EventType.JOURNEY_COMPLETE)?.occurredAt
+        val vehicleRegistration = journey.determineRegistrationsCombinedWithThoseFrom(thisJourneyEvents)
```

Comment:
Events are actually transient on the journey so we apply they to copy of the journey here and work out the registration to be applied to the persistent attribute on the journey below.